monitoring_service/server.py

```diff
@@ -93,45 +99,56 @@ def _run(self):
             EVENT_CHANNEL_SETTLED,
             lambda event, tx: self.on_channel_settled(event, tx)
         )
-        self.blockchain.add_confirmed_listener(
-            EVENT_TRANSFER_UPDATED,
-            lambda event, tx: self.on_transfer_updated(event, tx)
-        )
 
         # this loop will wait until spawned greenlets complete
         while self.stop_event.is_set() is False:
             tasks = gevent.wait(self.task_list, timeout=5, count=1)
             if len(tasks) == 0:
-                gevent.sleep(5)
+                gevent.sleep(1)
                 continue
             task = tasks[0]
+            log.info('%s completed (%s)' % (task, task.value))
```

Comment:
nitpick: I remember reading somewhere that we should use the style: `log.info('%s completed (%s)', task, task.value)`

Reply:
this will come when I switch over to `structlog` here too